pkg/orchestrator/pod.go

```diff
@@ -127,6 +127,10 @@ func ScrubContainer(c *v1.Container, scrubber *DataScrubber) {
 		}
 	}()
 
+	// scrub args
+	scrubbedArgs, _ := scrubber.ScrubSimpleCommand(c.Args)
```

Comment:
Will this work with no update done to the command scrubber code? I vaguely remember it was looking at the command name first but args may not contain any.